main.cpp

```diff
@@ -162,40 +186,35 @@ struct Piano
         double knobValue = 0.0;
     };
 
-    //adjust volume 
-    float pianoVolume(double volumeLevel, Knob volumeKnob);
-    //change sound
-    std::string chooseSound(bool pianoOn, bool organOn);
-    //make sound
+    void setVolume(double volumeKnobValue);
+    std::string displayChoosenSound(double soundKnobValue);
     double audio(int notePressed, double volume, bool footPedalPressed);
 };
 
-/*
-Thing 2) microwave oven
-5 properties:
-    1)  has door
-    2)  has timer
-    3)  power settings
-    4)  presets for different dishes
-    5)  has scale to measure weight
-3 things it can do:
-    1)  cook potato
-    2)  auto cook by weight
-    3)  change power setting
- */
+void Piano::setVolume(double volumeKnobValue)
+{
+    currentVolumeLevel = volumeKnobValue;
+}
+
+std::string Piano::displayChoosenSound(double soundKnobValue)
+{
+    if (soundKnobValue >= 50.0)
+    {
+        return pianoSound;
+    }
+    else
+    {
+        return organSound;
+    }
```

Comment:
If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
When the `if()` block is entered, the `return aValue;` is the last line of code to be executed.
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
`return` statements are the last thing that **ever happens** in a function
That means if your `if()` block is **NOT** entered, it will be skipped.  What is the next thing that happens?  `return anotherValue;`
that means you do **NOT** need the else block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    return anotherValue;
}
```
- [ ] revise